tests/results/DiffTest.py

```diff
@@ -93,6 +93,12 @@ def test_modified(self):
         self.uut.add_lines(0, ["0.1", "0.2"])
         self.uut.add_lines(1, ["1.1"])
         self.uut.change_line(3, "3", "3.changed")
+
+        # If delete is True then modified returns an empty list
+        self.uut.delete = True
+        self.assertEqual(self.uut.modified, [])
+        self.uut.delete = False
```

Comment:
can you move this into `test_delete`? Don't blow up unrelated test cases so much.